.circleci/config.yml

```diff
@@ -219,6 +219,7 @@ workflows:
           slack_channel: << pipeline.parameters.alerts-slack-channel >>
           context:
             - hmpps-common-vars
+          jdk_tag: "21.0"
```

Comment:
ref: https://github.com/ministryofjustice/crime-portal-gateway/blob/main/.circleci/config.yml#L125

Reply:
The job currently fails with a version mismatch like this: https://app.circleci.com/pipelines/github/ministryofjustice/court-case-matcher/3348/workflows/92e25145-1919-4171-a15b-e0ed79691e89/jobs/12295